.github/pull_request_template.md

```diff
@@ -37,9 +32,9 @@ Thank you for contributing! Please follow the steps below to help us process you
 
 ### Checklist
 I did the following to ensure that my changes were tested thoroughly:
-- [ ] __
-- [ ] __
+* __
+* __
```

Comment:
I find that I ever only need one bullet point, so maybe it makes sense to make it the default

```suggestion
- __
```